components/Player/PlayerBadges.jsx

```diff
@@ -1,66 +1,50 @@
 import React from 'react';
 import { connect } from 'react-redux';
-import ActionVisibility from 'material-ui/svg-icons/action/visibility';
 import ReactTooltip from 'react-tooltip';
 import Error from '../Error';
 import Spinner from '../Spinner';
 import { player } from '../../reducers';
 import styles from './PlayerHeader.css';
-import { IconCheese, IconSteam } from '../Icons';
+import { IconCheese, IconSteam, IconEye, IconEyeInactive } from '../Icons';
 
 export const PlayerBadgesIcons = ({ loading, error, cheese, tracked, steamLink }) => {
   const getPlayerBadges = () => {
     if (error) return <Error />;
     if (loading) return <Spinner />;
     return (
       <div className={styles.playerBadges}>
-        <div>
-          <a data-tip data-for="steam" className={styles.icon} href={steamLink} rel="noopener noreferrer" target="_blank">
-            <IconSteam
-              style={{
-                width: 16,
-                height: 16,
-                top: 8,
-                left: 4,
-              }}
-            />
+        <div data-tip data-for="steamLink" className={styles.iconButton}>
+          <a rel="noopener noreferrer" target="_blank" href={steamLink}>
+            <IconSteam className={styles.icon} />
+            <ReactTooltip id="steamLink" place="top" type="light" effect="float">
+              Steam profile
+            </ReactTooltip>
           </a>
-          <ReactTooltip id="steam" place="top" type="light" effect="float">
-            Steam profile
-          </ReactTooltip>
         </div>
-        {Math.round(new Date().getTime() / 1000.0) >= Number(tracked) ? null :
-          <div>
-            <div data-tip data-for="tracked">
-              <ActionVisibility className={styles.icon} />
-            </div>
+        {Math.round(new Date().getTime() / 1000.0) >= Number(tracked) ?
+          <div data-tip data-for="untracked" className={styles.iconButton}>
+            <IconEyeInactive className={styles.icon} style={{ height: 22, fill: 'darkgray' }} />
+            <ReactTooltip id="untracked" place="top" type="light" effect="float">
+              This user is untracked.
```

Comment:
This user is inactive, and replays of new matches will not be automatically parsed.